src/shared/constants/DeveloperUtils.js

```diff
@@ -0,0 +1,81 @@
+import { compareSync } from 'bcrypt';
+import { developerSchema } from '../../models/Developer';
+import { createDbConnection } from '../../services/database';
+import { isTest } from '../../config';
+
+const PROD_LIMIT = 2500;
+const FALLBACK_API_KEY = 'fallback_api_key';
+
+const determineLimit = (apiLimit) => (isTest ? apiLimit || PROD_LIMIT : PROD_LIMIT);
+const fetchRequestQuery = (req) => {
+  const { apiLimit } = req.query;
+  return apiLimit;
+};
+
+// const isSameDate = (first, second) =>
+//   first.getFullYear() === second.getFullYear() &&
+//   first.getMonth() === second.getMonth() &&
+//   first.getDate() === second.getDate();
+
+const isSameDate = (first, second) => {
+  const firstISO = first.toISOString().slice(0, 10); // Extract YYYY-MM-DD from first date
+  const secondISO = second.toISOString().slice(0, 10); // Extract YYYY-MM-DD from second date
+  return firstISO === secondISO;
+};
```

Comment:
please revert these change for this PR - if you want to make this change in a future PR with an explanation on why you are making this change to `isSameDate` then you can do so

Reply:
Done